### PR TITLE
Bug 1133146: Add geolocation capabilities.

### PR DESCRIPTION
**Pull request by pmclanahan**
Add functions for country lookup and MaxMind DB handling.

### CODE REVIEW

**Comment by pmclanahan** [comment]
@jgmize r?

**Comment by pmclanahan** [comment]
@jgmize bump. Jake's got the DB file ready and needs this in the repo so the new compiled `maxminddb` lib can be installed in the venvs.

File: bedrock/base/geo.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+from django.conf import settings
+
+try:
+    import maxminddb
+except ImportError:
+    maxminddb = None
+
+
+logger = logging.getLogger(__name__)
```
Comment (jgmize) [inline bedrock/base/geo.py]: FYI this causes a warning `No handlers could be found for logger "bedrock.base.geo"` for me locally

Comment (pmclanahan) [inline bedrock/base/geo.py]: Yeah. Our logging sucks. Just thought I'd add support for when it doesn't.
Can be removed for now if you'd prefer.

Comment (jgmize) [inline bedrock/base/geo.py]: Where do you expect the error logs to go in production?

File: bedrock/base/geo.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+try:
+    if maxminddb:
```
Comment (jgmize) [inline bedrock/base/geo.py]: Why is the `if` inside the `try`, instead of the other way around?

Comment (pmclanahan) [inline bedrock/base/geo.py]: Could be. I think it's just how the thing grew. I can rearrange if it'd make more sense.

File: bedrock/base/geo.py (hunk 1 above)
Comment (pmclanahan) [inline bedrock/base/geo.py]: No where at present. I just thought it better to be sending them than not.

Comment (jgmize) [inline bedrock/base/geo.py]: I'm a little concerned about the performance impact of the logging calls anyway, so if we don't have an immediate use for them I'd rather remove them.

File: bedrock/base/geo.py (hunk 2 above)
Comment (jgmize) [inline bedrock/base/geo.py]: It's a tiny nit, but if you don't mind :)

**Comment by pmclanahan** [comment]
@jgmize this is updated btw.

**Comment by jgmize** [comment]
Thanks, this lgtm now :)

**Pull request merged**